code/game/objects/items/weapons/material/shards.dm

```diff
@@ -68,7 +68,7 @@
 		if(ishuman(M))
 			var/mob/living/carbon/human/H = M
 
-			if(H.species.siemens_coefficient<0.5 || isvaurca(H) || (H.species.flags & (NO_EMBED))) //Thick skin.
+			if(H.species.siemens_coefficient<0.5 || isunathi(H) || isvaurca(H) || (H.species.flags & (NO_EMBED))) //Thick skin.

```

Comment:
excuse me.

This is not a documented change.

Reply:
![image](https://user-images.githubusercontent.com/17379794/52526823-f57d6900-2ccf-11e9-96d5-4dfecda632c4.png)

Reply:
Still, isnt changelogged

Reply:
handled